setup.py

```diff
@@ -1,67 +0,0 @@
-import codecs
-import setuptools
-import re
-import ast
-
-_version_re = re.compile(r'__version__\s+=\s+(.*)')
-
-with open('src/icalendar/__init__.py', 'rb') as f:
-    version = str(ast.literal_eval(_version_re.search(
-        f.read().decode('utf-8')).group(1)))
-
-
-shortdesc = 'iCalendar parser/generator'
-longdesc = ''
-for fname in ['README.rst', 'CONTRIBUTING.rst', 'CHANGES.rst', 'LICENSE.rst']:
-    with codecs.open(fname, encoding='utf-8') as f:
-        longdesc += f.read()
-        longdesc += '\n'
-
-tests_require = []
-install_requires = [
-    'python-dateutil',
-    # install requirements depending on python version
-    # see https://www.python.org/dev/peps/pep-0508/#environment-markers
-    'backports.zoneinfo; python_version < "3.9"',
-    'tzdata'
-]
-
-
-setuptools.setup(
-    name='icalendar',
-    version=version,
-    description=shortdesc,
-    long_description=longdesc,
-    long_description_content_type="text/x-rst",
-    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
-        'Development Status :: 3 - Alpha',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: BSD License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 3.8',
-        'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3.10',
-        'Programming Language :: Python :: 3.11',
-        'Programming Language :: Python :: 3.12',
-        'Programming Language :: Python :: Implementation :: CPython',
-        'Programming Language :: Python :: Implementation :: PyPy',
-    ],
-    keywords='calendar calendaring ical icalendar event todo journal '
-             'recurring',
-    author='Plone Foundation',
-    author_email='plone-developers@lists.sourceforge.net',
```

Comment:
@mauritsvanrees, the setup.py file states a plone developers email for the author. I would keep this for now or should we change this. Is it still in use?

Reply:
The last message to this list was in 2021.

https://sourceforge.net/p/plone/mailman/plone-developers/

I'd say change it, as you need to subscribe to post to that list anyway. I think it is fine to use your own email, or request an email address from the Plone Foundation, such as icalendar@plone.org or nicco@plone.org. They use Google Workspace for Nonprofits, so there is no cost, other than the setup effort.

Reply:
Indeed the address is basically unused, and it only lingers in several `setup.py` files like this one.  I usually leave it, because I don't know a better one. :-). It is fine to change.

Reply:
Hm. This email seems to be very much of no value. I will leave it with a comment. Thanks!